Description: OpenMP lib fails to compile under gfortran. Leave out omp support for now.
Author: Alastair McKinstry <mckinstry@debian.org>
Forwarded: no
Last-Updated: 2018-11-29

Index: flang-20180921/runtime/flang/CMakeLists.txt
===================================================================
--- flang-20180921.orig/runtime/flang/CMakeLists.txt
+++ flang-20180921/runtime/flang/CMakeLists.txt
@@ -147,7 +147,6 @@ SET(FTN_INTRINSICS_DESC_INDEP
   merge.c
   mvbits3f.c
   nargs3f.c
-  omp_lib.F95
   outstr3f.c
   packtimeqq3f.c
   perror3f.c
@@ -191,6 +190,10 @@ SET(FTN_INTRINSICS_DESC_INDEP
   wait3f.c
   )
 
+if (CMAKE_Fortran_COMPILER_ID MATCHES "flang")
+  LIST(APPEND FTN_INTRINSICS_DESC_INDEP omp_lib.F95)
+endif()
+
 SET(FTN_INTRINSICS_DESC_DEP
   red.c
   red_all.c
